### PR TITLE
Merged css

### PR DESCRIPTION
**Pull request by BlakeDonn**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Styling- no new features

### Detailed Description

* Added some CSS animation into the project for styling and fun

### Why is this change required? What problem does it solve?

* it was added for styling into the project

### Were there any challenges that arose while implementing this feature? If so, how were the addressed?

* this was added just for styling

### Files modified:
- [x] index.html
- [x] styles.css
- [ ] scripts.js
- [ ] activity.js
- [ ] hydration.js
- [ ] sleep.js
- [ ] users.js
- [ ] testSuite.js

### CODE REVIEW

**Comment by BlakeDonn** [comment]
All conflicts were done as a group

File: src/Activity.js [hunk 1]
```diff
@@ -13,14 +13,11 @@ class Activity {
   getUserData(id) {
     return userRepository.returnUserData(id);
   }
-
-
```
Comment (MichaelEWalker87) [inline src/Activity.js:17]: delete dead space

File: src/scripts.js [hunk 2]
```diff
@@ -154,6 +153,18 @@ function compareDayActivity() {
...
+function displayConsecutiveDays() {
+  let consecutiveActivityDays = document.querySelector('.consecutive-days')
+  let activityConsecutiveDays = activity.consecutiveDays(user.userData.id)
+  activityConsecutiveDays.forEach((day, index) => {
+    consecutiveActivityDays.innerHTML +=
+    `<ul> ${activityConsecutiveDays[index]}</ul>`
+  })
+}
+
+
```
Comment (MichaelEWalker87) [inline src/scripts.js:165]: delete dead space

File: src/styles.css [hunk 3]
```diff
@@ -161,3 +172,120 @@
   right: -80px;
   background-color: #4AACC5;
 }
+
+
+
```
Comment (MichaelEWalker87) [inline src/styles.css:177]: delete this dead space

**Pull request merged**